src/status_im/ui/screens/chat/input/input.cljs

```diff
@@ -158,23 +158,29 @@
   (letsubs [username           [:contacts/contact-name-by-identity from]
```

Comment:
I can't make suggestions on this one since you didn't change it so here what it should look like:

```
(defview reply-message [alias {:keys [text name]]
```